src/pages/EntityDetails/_entity-details.scss

```diff
@@ -336,8 +336,31 @@
     }
   }
 
-  .entity-details__machines,
-  .entity-details__relations {
+  .entity-details__integrations {
+    th,
+    td {
+      // Type.
```

Comment:
One small nitpicky remark: In the previous PRs (#1657 and #1658) the names of the columns didn't have a period at the end. Was wondering if maybe we can standardize this and follow one way of writing these comments?

Reply:
Thanks. We typically end comments with a period, so I've updated the comments from the other PRs.